boards/shields/adafruit_16_ch_pwm_servo/Kconfig.defconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022 Nick Ward <nix.ward@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_ADAFRUIT_16_CH_PWM_SERVO
+
+config I2C
+	default y
+
+config PWM_PCA9685
+	default y
+
+endif # SHIELD_ADAFRUIT_16_CH_PWM_SERVO
```

Comment:
this is redundant, please check how drivers use DT_HAS* and select

Reply:
Removed